plugins/backstage-plugin-devcontainers-backend/package.json

```
@@ -44,5 +45,10 @@
   },
   "files": [
     "dist"
+  ],
+  "keywords": [
+    "backstage",
+    "dev containers",
+    "devcontainers"
```

I don't know that these keywords should have spaces, but perhaps they can have hyphens?
```suggestion
  "keywords": [
    "backstage",
    "devcontainers",
    "github",
    "gitlab",
    "bitbucket",
    "developer-tools",
    "ide",
    "vscode"
```
```

---

@Parkreiner I just saw your note about not wanting to spam keywords and feel that's reasonable. I think some important keywords to include are those that are vendor-specific, i.e. "coder", "vscode", "github", "gitlab", and "bitbucket". 
The others I feel less strongly about!